### DISCUSSION TITLE
"Failed to fetch Workbook Data Sources" Error in Tableau Connection

### DISCUSSION THREAD

**Discussion by herykeven** [open-metadata/OpenMetadata#13889] [Q&A]
Hello everyone,

Testing my dashboard connection with tableau, ive encountered an issue while trying to extract data models from Tableau, and im seeking assistance to resolve it. The error message im encountering is:

**"Failed to fetch Workbook Data Sources, please validate if the user has access to fetch Data Sources, and Metadata API is enabled for Tableau Server."**
![image](https://github.com/open-metadata/OpenMetadata/assets/78233037/9bb41820-809b-45c5-be2f-0e2145b6f83f)

However, in my opinion, its not possible that the metadata api is not enabled, since i am having success in all the previous steps of the connection test. Im using a Tableau user with all the necessary permissions. Despite this, im still facing this error.

I have thoroughly reviewed the permissions, but i can't seem to identify the root cause. Has anyone else encountered this issue or have insights on how to troubleshoot it?

ty

**Comment by SChalupaBraiins** [comment]
I have the very same symptoms, has anybody solved this yet?

**Comment by Prajwal214** [comment]
@herykeven Can you share the tableau connection config you are using? You can mask the sensitive info.

**Comment by AAA97AAA** [comment]
Hi, I had the same issue. 
In the tableau connection config, when adding the API version, the warning no longer appears. 

<img width="149" height="75" alt="image" src="https://github.com/user-attachments/assets/4f6ad55a-07c2-4617-be57-268cb9212235" />

Tableau API version can be found at: https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_concepts_versions.htm#version_and_rest
